### PR TITLE
Condensed Capsaicin has more noticeable effects - more interactions with milk

### PR DESCRIPTION
**Pull request by wesoda25**
## About The Pull Request

condensed capsaicin hits a little harder now - however its effects are null if the user drank milk

## Why It's Good For The Game

i was doing a spicy food gimmick as chef and was kinda disappointed with spicy code, so spiced it up a little

## Changelog
:cl:
tweak: condensed capsaicin is spicier now
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/food_reagents.dm [hunk 1]
```diff
@@ -296,10 +296,21 @@
...
 			addtimer(CALLBACK(victim, /mob.proc/remove_movespeed_modifier, MOVESPEED_ID_PEPPER_SPRAY), 10 SECONDS)
 		victim.update_damage_hud()
+	if(method == INGEST)
+		if(!holder.has_reagent(/datum/reagent/consumable/milk))
+			if(prob(15))
+				to_chat(M, "<span class='danger'>[pick("Your head pounds.", "Your mouth feels like its on fire.", "You feel dizzy.")]</span>")
+			if(prob(10))
+				victim.blur_eyes(1)
+			if(prob(10))
+				victim.Dizzy(1)
```
Comment (ATH1909) [inline code/modules/reagents/chemistry/reagents/food_reagents.dm]: will this have any noticeable effect?

Comment (wesoda25) [reply code/modules/reagents/chemistry/reagents/food_reagents.dm]: yes

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Instead of explicitly checking for milk, why not make a reaction that neutralizes both?

**Comment by wesoda25** [comment]
> Instead of explicitly checking for milk, why not make a reaction that neutralizes both?

Milk already purges capsaicin. I just want the symptoms to be quelled too so its a little more noticeable IC. (idk if this is what you meant by this but ill assume so)

Off topic question here though, one thing I tried to do was change milks taste var if the user has capisaicin in them, but it didnt work. Any idea how to do it?

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**